docs/content/preview/explore/multi-region-deployments/asynchronous-replication-ysql.md

```diff
@@ -16,165 +16,115 @@ menu:
 
 1. Open `ysqlsh` for "Data Center - West" by specifying the host IP address of `127.0.0.2`, as follows:
 
-     ```sh
-   ./bin/ysqlsh -h 127.0.0.2
-   ```
+    ```sh
+    ./bin/ysqlsh -h 127.0.0.2
+    ```
 
 1. Create the table `users`, as follows
 
-     ```sql
-   CREATE TABLE users (
-       email varchar(35) PRIMARY KEY,
-       username varchar(20)
-       );
-   ```
+    ```sql
+    CREATE TABLE users (
+      email varchar(35) PRIMARY KEY,
+      username varchar(20)
+      );
+    ```
 
 Having two identical tables on your clusters allows you to set up xCluster replication across two data centers.
 
-
 ## Configure unidirectional replication
 
-To configure "Data Center - West" to be the target of data changes from the "Data Center - East" cluster, you need to use the `yb-admin` `setup_universe_replication` command. Review the syntax and run the following command:
+To configure "Data Center - West" to be the target of data changes from the "Data Center - East" cluster, you need to use the [yb-admin](../../../admin/yb-admin/) utility [setup_universe_replication](../../../admin/yb-admin/#xcluster-replication-commands) command. The syntax is as follows:
 
-```sh
+```sh.output
 yb-admin -master_addresses <target-master-addresses> \
-setup_universe_replication <source-universe_uuid> <source_master_addresses> <source-table-ids>
+    setup_universe_replication <source-universe_uuid> \
+    <source_master_addresses> <source-table-ids>
 ```
 
-- *target-master-addresses*: a comma-separated list of the YB-Master servers. For this simulation, you have one YB-Master server for each cluster (typically, there are three).
-- *source-universe-uuid*: a unique identifier for the source cluster. The UUID can be found in the YB-Master UI (`<yb-master-ip>:7000`).
-- *source-table-ids*: A comma-separated list of `table_id` values. The generated UUIDs can be found in the YB-Master UI (`<yb-master-ip>:7000`/tables).
+- *target-master-addresses*: a comma-separated list of the target YB-Master servers. For this simulation, you have one YB-Master server for the target, 127.0.0.2:7100.
+- *source-universe-uuid*: a unique identifier for the source cluster. Look up the UUID in the source YB-Master UI (<http://127.0.0.1:7000>).
+- *source-master-addresses*: a comma-separated list of the source YB-Master servers. For this simulation, you have one YB-Master server for the source, 127.0.0.1:7100.
+- *source-table-ids*: A comma-separated list of table UUIDs. For this simulation, the `users` table; look up the UUID in the YB-Master UI (<http://127.0.0.1:7000/tables>).
 
-Based on your actual values (which you obtained from the YB-Master UI page at `yb-master-ip>:7000`), run the `yb-admin` `setup_universe_replication` command similar to the one shown in the following example:
+Based on actual values you obtained from the YB-Master UI, run the yb-admin `setup_universe_replication` command similar to the one shown in the following example:
```

Comment:
Mention that the following yb-admin command must be run from the user's Yugabyte home directory from another terminal , because before this step, there are two ysqlsh terminals with tables created.